Xamarin.Forms.Controls/GalleryPages/CollectionViewGalleries/CarouselViewGalleries/CarouselViewGallery.cs

```diff
@@ -34,19 +34,25 @@ public CarouselViewGallery()
 						GalleryBuilder.NavButton("CarouselView (XAML, Horizontal)", () =>
 							new CarouselXamlGallery(), Navigation),
 						GalleryBuilder.NavButton("CarouselView (Indicators Forms)", () =>
-							new CarouselItemsGallery(false,false,false), Navigation),
+							new CarouselItemsGallery(false,false,false,false,false,true), Navigation),
```

Comment:
Can we make all these constructor parameters optional with a default value of `false`? It's pretty hard to tell what's being tested here.

Reply:
fixed @hartez